### DISCUSSION TITLE
ModuleNotFoundError: No module named 'ctvbot'

### DISCUSSION THREAD

**Discussion by sezeralaca** [KevinBytesTheDust/CVAmp#216] [Q&A]
Hi, I'm trying to launch code from VSCode but I keep getting this error. Thanks in advance

PS C:\Users\isten\Desktop\crude-twitch-viewer-bot-0.6.1> & C:/Users/isten/AppData/Local/Programs/Python/Python310/python.exe c:/Users/isten/Desktop/crude-twitch-viewer-bot-0.6.1/ctvbot/sites.py
Traceback (most recent call last):
  File "c:\Users\isten\Desktop\crude-twitch-viewer-bot-0.6.1\ctvbot\sites.py", line 8, in <module>
    from ctvbot import utils
ModuleNotFoundError: No module named 'ctvbot'

**Comment by jlplenio** [comment]
You can try to set the cwd to the root folder, i.e. crude-twitch-viewer-bot-0.6.1.
